shared_model/builders/protobuf/builder_templates/transaction_template.hpp

```diff
@@ -185,12 +200,18 @@ namespace shared_model {
         return addSignatoryRaw(account_id, public_key.hex());
       }
 
+      auto addSignatory(const interface::types::AccountIdType &account_id,
+                        interface::types::PublicKeyHexStringView public_key)
+          const {
+        return addSignatoryRaw(account_id, public_key);
+      }
+
       auto removeSignatoryRaw(const interface::types::AccountIdType &account_id,
-                              const std::string &public_key) const {
+                              std::string_view public_key) const {
```

Comment:
```suggestion
                              interface::types::PublicKeyHexStringView public_key) const {
```

Reply:
The idea of raw method is to be able to set an arbitrary string. See https://github.com/hyperledger/iroha/pull/409/files/bc773a1c50252cf0234c08ebd044267edcdc504b..0d2768d26c9c6595fb506345c4a3f4ce6aba98fd#diff-18924faa65b2ff1ff779b7c27c051252R224-R228